### ISSUE TITLE
Installation issues: 'there is no package called memo'

### ISSUE THREAD

**Issue by alexanderbates** [natverse/fafbseg#81]
```
Downloading GitHub repo natverse/fafbseg@HEAD
✔  checking for file ‘/tmp/RtmpDA38AF/remotes3e6f3eb36878/natverse-fafbseg-eba061a/DESCRIPTION’
─  preparing ‘fafbseg’:
✔  checking DESCRIPTION meta-information
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
   Removed empty directory ‘fafbseg/vignettes’
─  building ‘fafbseg_0.9.0.9000.tar.gz’
   
Installing package into ‘/home/travis/R/Library’
(as ‘lib’ is unspecified)
* installing *source* package ‘fafbseg’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(name) : there is no package called ‘memo’
Error: unable to load R code in package ‘fafbseg’
Execution halted
ERROR: lazy loading failed for package ‘fafbseg’
* removing ‘/home/travis/R/Library/fafbseg’
* restoring previous ‘/home/travis/R/Library/fafbseg’
Error: Failed to install 'fafbseg' from GitHub:
  (converted from warning) installation of package ‘/tmp/RtmpDA38AF/file3e6f32222185/fafbseg_0.9.0.9000.tar.gz’ had non-zero exit status
Execution halted
The command "Rscript -e 'deps <- remotes::dev_package_deps(dependencies = NA);remotes::install_deps(dependencies = TRUE);if (!all(deps$package %in% installed.packages())) { message("missing: ", paste(setdiff(deps$package, installed.packages()), collapse=", ")); q(status = 1, save = "no")}'" failed and exited with 1 during .
```

**Comment by jefferis** [comment]
Ok you're right that this new dependency is suggested but not used conditionally in all places. If you install from scratch or use natmanager::install it will work. The problem is that remotes does not install new suggests packages by default.

**Comment by alexanderbates** [comment]
FYI, it was causing Travis to throw an error, e.g. for hemibrainr here: https://travis-ci.com/github/natverse/hemibrainr/builds/214032062

**Comment by jefferis** [comment]
> FYI, it was causing Travis to throw an error, e.g. for hemibrainr here: https://travis-ci.com/github/natverse/hemibrainr/builds/214032062
@alexanderbates if you read the logs you will see that the hemibrainr failure is unrelated. There are two issues with the hemibrainr build
1. a NOTE that needs fixing:
```
nlscan_split: possible error in plot3d_split(someneuronlist[i], col =
  col, WithConnectors = WithConnectors, WithNodes = WithNodes, highflow
  = highflow, radius = radius, prepost = prepost, splitnode =
  splitnode, WithNodes = WithNodes, soma = soma, soma.alpha =
  soma.alpha, transmitters = transmitters, transmitter.alpha =
  transmitter.alpha, lwd = lwd, check.template = check.template, ...):
  formal argument "WithNodes" matched by multiple actual arguments
````
2. a **deploy** error. since the move to the natverse org there is a probably with the ssh key used to deploy the pkgdwown to github:
```
Deploying application
Error: No deploy key found, please setup with `travis::use_travis_deploy()`
Execution halted
Script failed with status 1
```
it would be great if you could follow this up.

**Comment by alexanderbates** [comment]
I noticed those as well, and fixed the argument. Though travis does not install for my R version:

```
package ‘travis’ is not available (for R version 3.6.2)
```

And I didn't look into a workaround yet, but am looking here: https://github.com/ropensci-archive/travis
